README.md

```diff
@@ -2,20 +2,30 @@
 
 The explainer outlines the motivating use cases for the API, its key features, and how it can benefit web developers and users alike.
 
 ## Motivating Use Cases
 
-### Augmenting `window.print()`
+Although `window.print()` offers basic printing functionality through the browser's default print dialog, it is a relatively simple API. Developers' capabilities are restricted to selecting a file for printing and triggering the print dialog, which places the decision-making responsibility solely on the end user. To address these limitations and facilitate advanced use cases, we aim to provide developers with greater control over the printing process and expand the range of options available to them.
 
-While `window.print()` is a basic function that triggers the browser's default print dialog, the `navigator.printing` API provides a more comprehensive set of methods and options for querying local printers, submitting print jobs with customization options, managing print jobs, and customizing print workflows. This extended functionality allows developers to have more control over the printing process and enables advanced use cases, such as printing specific pages or sections of a document, setting print options like paper size and orientation, and monitoring the status of print jobs.
+### Customizing Print Settings
+The enhanced features aim to give developers greater influence over the printing process of a document. These capabilities include selecting the printer without requiring user input, printing specific pages or sections of the document, and adjusting print settings such as paper size, orientation, or finishings directly through the script.
 
-### Leveraging Progressive Web Apps
-The new Web API can motivate developers to switch to Progressive Web Apps (PWAs) by offering enhanced printing capabilities that were traditionally limited to native applications (in particular, support for Printer Redirection).
+### Knowing Your Printers
+To enable customization of settings, developers need to have knowledge about the capabilities of the printer. The API provides a convenient method for accessing this information, which enhances the printing experience by allowing developers to retrieve relevant details about what the printer can and cannot do.
+
+### Canceling & Tracking Print Jobs
+When users utilize `window.print()`, they have limited visibility and control over the printing process. However, the API fills this gap by offering the ability to track the progress of print jobs and allowing users to cancel them if needed. This functionality provides users with a way to monitor and influence the state transitions of their print jobs, enhancing their overall printing experience.
```

Comment:
Is this true? Browser and/or operating system UI typically give the user visibility into print job status.

Reply:
Oh, that comes as an actual surprise -- I thought we only get a notification when a print job is created! Indeed, there's a special [class](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:chrome/browser/ash/printing/cups_print_job_notification.h) that displays notifications when print job state changes.

I've rewritten this piece to underline the positive impact of having all job status data processed in one place.